lib/ProductOpener/Products.pm

```diff
@@ -75,6 +75,7 @@ BEGIN
 		&server_for_product_id
 		&data_root_for_product_id
 		&www_root_for_product_id
+		&split_code
```

Comment:
It's already exported here: https://github.com/openfoodfacts/openfoodfacts-server/blob/89119c8f92ee67120b95f1037c8ad453c1929a55/lib/ProductOpener/Products.pm#L73

Reply:
Thanks